src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs

```diff
@@ -27,6 +27,7 @@ public class DiagnosticTest : RemoteExecutorTestBase
 
         [Fact]
         [ActiveIssue("dotnet/corefx #17925", TargetFrameworkMonikers.NetFramework)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot)] // Internals reflection not supported on uapaot
```

Comment:
I'm not famliar with these tests so I don't know what internal reflection is happening. But is there any value we are losing here -- can some of the tests be salvaged, eg., by skipping something (?in CollectStatisticsDiagnostics) in UAPAOT?